src/main/java/seedu/duke/Duke.java

```diff
@@ -4,11 +4,48 @@
 import seedu.duke.logic.parser.Parser;
 import seedu.duke.ui.Ui;
 
+import java.util.ArrayList;
+
 public class Duke {
 
     private static Parser parser;
+    private static ArrayList<Bus> busesInZoneA = new ArrayList<>();
+    private static final String[] ZONE_A = {"AA1", "AA2"};
```

Comment:
String array declared but not use.
Could be implemented below replacing in line 46: Instead of "AA1", could add Zone_A[0] and Zone_A[1] to show a sequential approach in preparation for larger number of bus routes. Keep in view for future commits.